openapi/components/schemas/Common/CommonOrder.yaml

```
@@ -80,6 +80,14 @@ properties:
     description: Date and time when the order is voided.
     allOf:
       - $ref: ../ServerTimestamp.yaml
+  abandonTime:
+    type: string
+    description: |-
+      Date and time when pending order is automatically voided if the value is set.
+      When the property is not passed during creation then uses a [pending order TTL](https://user-api-docs.rebilly.com/tag/Organizations/operation/PatchOrganization#!path=settings/billing/pendingOrderTtl&t=request)
+      setting to calculate value.
```

suggestion: I think we can leave out 'if the value is set.'

I'm assuming we are referring to the `abandonTime` value

```suggestion
      Date and time when the pending order is automatically voided.
      If this value is not passed during order creation, 
      a [pending order TTL](https://user-api-docs.rebilly.com/tag/Organizations/operation/PatchOrganization#!path=settings/billing/pendingOrderTtl&t=request) setting is used to calculate the value.
```